gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -82,6 +82,10 @@
 #include "MemcheckWrapper.hpp"
 #endif /* defined(OMR_VALGRIND_MEMCHECK) */
 
+struct memorySizes{
```

Comment:
Let's call it HeapSizes.
Use uintptr_t.
Move the structure declaration  just before the callback that it's using.